### PR TITLE
💫 [IMPR] Add sources websites and pictograms to Aggregator (refs #3569)

### PR DESCRIPTION
**Pull request by Chatewgne**
## Checklist

- [x] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/contribute/contributing.html)
- [x] My code respects the Definition of done available in the [Development section of the documentation](https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes
- [x] I added an entry in the changelog file
- [x] My commits are all using prefix convention (emoji + tag name) and references associated issues
- [x] I added a label to the PR corresponding to the perimeter of my contribution
- [x] The title of my PR mentionned the issue associated

### CODE REVIEW

File: geotrek/trekking/tests/test_parsers.py [hunk 1]
```diff
@@ -428,12 +433,12 @@ def content(self):
...
         trek = Trek.objects.all().first()
         self.assertEqual(Attachment.objects.filter(object_id=trek.pk).count(), 3)
-        self.assertEqual(Attachment.objects.first().attachment_file.read(), b'11')
+        self.assertEqual(Attachment.objects.first().attachment_file.read(), b'15')
         call_command('import', 'geotrek.trekking.tests.test_parsers.TestGeotrekTrekParser', verbosity=0)
         self.assertEqual(Trek.objects.count(), 5)
         trek.refresh_from_db()
         self.assertEqual(Attachment.objects.filter(object_id=trek.pk).count(), 3)
-        self.assertEqual(Attachment.objects.first().attachment_file.read(), b'13')
+        self.assertEqual(Attachment.objects.first().attachment_file.read(), b'25')
```
Comment (Chatewgne) [inline geotrek/trekking/tests/test_parsers.py:441]: Adding some requests made line 424 above be called a few more times

**Review by babastienne — APPROVED** [review]

**Review by juggler31 — APPROVED** [review]

File: geotrek/common/parsers.py [hunk 2]
```diff
@@ -1477,6 +1481,34 @@ def next_row(self):
...
+                name = result['name']
+                pictogram_url = result['pictogram']
+                website = result['website']
+                source, created = RecordSource.objects.update_or_create(**{'name': name}, defaults={'website': website})
+                if created:
+                    self.add_warning(_(f"Source '{name}' did not exist in Geotrek-Admin and was automatically created"))
+                if not pictogram_url and source.pictogram:
+                    source.pictogram.delete()
+                elif pictogram_url:
+                    pictogram_filename = os.path.basename(pictogram_url)
```
Comment (marcantoinedupre) [inline geotrek/common/parsers.py:1498]: I do not understand the URL to filename conversion. When I test locally:

```
>>> os.path.basename("https://foo.bar/hello/world/")
''
```

Comment (Chatewgne) [reply geotrek/common/parsers.py:1498]: Pictogram URLs include the filename:
```
>>> os.path.basename("https://foo.bar/hello/world/afile.txt")
'afile.txt'
```

Comment (marcantoinedupre) [reply geotrek/common/parsers.py:1498]: Alright :+1:

**Pull request merged**